### ISSUE TITLE
error occur when load network

### ISSUE THREAD

**Issue by xiaosuzhang** [ayooshkathuria/YOLO_v3_tutorial_from_scratch#8]
I train my yolov3 model using my own datasets and get the .weight file. When i use the darnet.py to load my .weights, some error is occur as followed:

RuntimeError: invalid argument 2: size '[18 x 256 x 1 x 1]' is invalid for input with 4607 elements at ..\src\TH\THStorage.c:41

my pytorch verson is 0.4.0 on windows.
how should i do to solve the problem?
Tnanks?

**Comment by xiaosuzhang** [comment]
My classes=1 and the anchors boxs per cell is 3. And I also change the values of variable variable num_classes,and set the num_classes=1.

**Comment by ayooshkathuria** [comment]
Are you using the same config file cfg/yolov3.cfg?

**Comment by xiaosuzhang** [comment]
Yes! I use the yolov3.cfg based on the the model from https://pjreddie.com/darknet/yolo/,  only change the num_classes=1 and filters=18 to train on my own dataset.

**Comment by ayooshkathuria** [comment]
That is strange indeed. Something is not quite right with the code. Can you fork the code, and host it on your account. I could then pull your code, and look what's up with the implementation later today. or over the weekend. It would be nice if you uploaded your weights file somewhere too, so i can have the code exactly as same as you.

**Comment by ayooshkathuria** [comment]
Also, what are you trying to detect? You need to provide namesfile instead of coco.names, containing the name of the class you want to detect.

**Comment by xiaosuzhang** [comment]
can i send the mail to you??

**Comment by ayooshkathuria** [comment]
Mail would be all right, but if you fork and host a repo on your account, I can simply issue the fix as a pull request which you can merge. 

Do you have any issues with the process of forking, and committing changes, like if you're new to git, I can help. It's better you learn to embrace it coz these skills are very handy. I highly recommend making your own repo so i can pull it. 

Otherwise, kathuria.ayoosh@gmail.com is where you wanna send it. :)

**Comment by xiaosuzhang** [comment]
Yes i am a newer to git, and i will learn how to make my own repo. Thank you very much!!

**Comment by ayooshkathuria** [comment]
First you go to this repo, and hit the "Fork" button at the top left. This will create a copy of this repo on your own account. 

Then, copy the url of **your** repo (called your fork on my repo), and clone it on your computer. make the changes. Then, do something like this. 
```
git add . 
git commit -m "Changes for custom dataset"
git push origin master
```
Then the changes would be uploaded to your repo and I can then pull them. I'd highly recommend you to invest a day or two learning about git, as it makes life much easier when you have to adapt other's work for our own purposes. Ping here when you have pushed all the changes.

**Comment by ayooshkathuria** [comment]
Here's a link: https://tutorialzine.com/2016/06/learn-git-in-30-minutes

**Comment by xiaosuzhang** [comment]
Thank you very much.

**Comment by xiaosuzhang** [comment]
I have created the new respository and upload the cfg file and the .name file. The i will send the weights file  to your email address because the weights file is very big.

**Comment by ayooshkathuria** [comment]
Don't mail me the weights file since mail attachments have size limit. Upload it on Google drive maybe and gimme the link.

**Comment by xiaosuzhang** [comment]
OK

**Comment by xiaosuzhang** [comment]
the Google drive link of my files:
https://drive.google.com/open?id=1EFz-BuPdvyNm0RzIDgOkxlfp4hQhhHqN
Thank you again

**Comment by ayooshkathuria** [comment]
All right. Gimme a few days to look at it. I'll try to make it work.

**Comment by LittleSunH** [comment]
I meet the same question ,did you solve it ?

**Comment by TiongSun** [comment]
https://github.com/ayooshkathuria/pytorch-yolo-v3/issues/19
